CSharpBasics/CSharpBasics/Program.cs

```diff
@@ -0,0 +1,189 @@
+﻿new Homework_2().Task_1();
+
+class Homework_2
+{
+    public void Task_1()
+    {
+        Console.WriteLine("Enter your age");
+        int age = Convert.ToInt32(Console.ReadLine());
+
+        Console.WriteLine("Enter your BMI");
+        double bodyMassIndex = Convert.ToDouble(Console.ReadLine());
+
+        Console.WriteLine("Enter your blood group");
+        char bloodGroup = Convert.ToChar(Console.ReadLine());
+
+        Console.WriteLine("Do you smoke? (true or false)");
+        bool smokingStatus = Convert.ToBoolean(Console.ReadLine());
+
+        Console.WriteLine("Your age:" + age);
+        Console.WriteLine("Your BMI:" + bodyMassIndex);
+        Console.WriteLine("Your blood group" + bloodGroup);
+        Console.WriteLine("You smoke: " + smokingStatus);
+    }
+
+    public void Task_2()
+    {
+        Console.WriteLine("In which year Waterloo battle took place?");
+        int answer1 = int.Parse(Console.ReadLine());
+
+        Console.WriteLine("Did Napoleon win? (True or False):");
+        bool answer2 = bool.Parse(Console.ReadLine());
+
+        Console.WriteLine("Year of the Waterloo battle: " + answer1);
+        Console.WriteLine("Napoleon was the winner: " + answer2);
+    }
+    public void Task_3()
+    {
+        Console.WriteLine("Enter an integer:");
+        string input = Console.ReadLine();
+        bool output = int.TryParse(input, out int number);
+        if (output == true)
```

Comment:
this could be simplified :)
if (output)
            Console.WriteLine($"Correct. The integer is {number}");
        else
            Console.WriteLine("Error");